.github/workflows/pythonpackage.yml

```diff
@@ -14,141 +14,19 @@ jobs:
     - uses: actions/checkout@v2
```

Comment:
I would also be OK to rename this test to something more suitable like `unit-test.yml`